pep-0008.txt

```diff
@@ -1040,27 +1064,27 @@ Any backwards compatibility guarantees apply only to public interfaces.
 Accordingly, it is important that users be able to clearly distinguish
 between public and internal interfaces.
 
-Documented interfaces are considered public, unless the documentation
-explicitly declares them to be provisional or internal interfaces exempt
-from the usual backwards compatibility guarantees. All undocumented
-interfaces should be assumed to be internal.
+* Documented interfaces are considered public, unless the documentation
+  explicitly declares them to be provisional or internal interfaces exempt
+  from the usual backwards compatibility guarantees. All undocumented
+  interfaces should be assumed to be internal.
```

Comment:
In my initial draft, I was originally going to put the note in this section, then realised it made more sense in the naming section.